k2/csrc/fsa_utils.cu

```diff
@@ -99,75 +99,237 @@ static void SplitStringToVector(const std::string &in, const char *delim,
   }
 }
 
-// Create an acceptor from a stream.
-static Fsa AcceptorFromStream(std::string first_line, std::istringstream &is,
-                              bool openfst) {
+/* Create an acceptor from a stream, assuming the acceptor is in the K2 format:
```

Comment:
In the original design without taking openfst into account, Dan wanted
a version accepting a parameter `negate_scores`. I am not sure
whether the parameter is still needed here.

Reply:
@danpovey do we want to have the option for negating score when openfst == false? Or do we only want that when openfst == true?